docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,311 @@
+  ```
+
+# Sender
+
+  In order to send a payjoin transaction, the sender need to complete two steps.
+
+  First, obtains the reciever BIP21 URI(usually via qr-code), construct a PSBT
+  based on paramaters, like address, payjoin endpoint(pj) and amount, as
+  specified in the URI.
+
+  Second, the sender construct an HTTPs request, with the PSBT attached to the
+  body. After sending the request, the sender waits for the response and when it
+  arrives, its verified, signed and broadcasted to the bitcoin blockchain.
+
+
+  Lets start implementing the sender functionality inside
+  `payjoin-tutorial/src/main.rs`
+
+  * Add imports:
+
+  ```rust
+    use bitcoincore_rpc::RpcApi;
+    use payjoin::bitcoin::address::NetworkChecked;
+    use std::collections::HashMap;
+    use std::str::FromStr;
+
+    use bitcoincore_rpc::bitcoin::psbt::Psbt;
+    use bitcoincore_rpc::bitcoin::Txid;
+    use payjoin::base64;
+    use payjoin::send::RequestBuilder;
+   ```
+
+  * Add an `https_agent` function. 
+
+    As you would guess, the `https_agent` function is used in order to make
+    https requests, the sender will use this function to send the payjoin
+    request to the reciever.
+
+    Here, we are using `danger_accept_invalid_certs` as we running this locally,
+    in production this should be initiated with valid certificates only.
+
+```rust
+ fn https_agent() -> reqwest::Client {
+     let https = reqwest::Client::builder()
+         .danger_accept_invalid_certs(true)
+         .build()
+         .expect("failed to build https client");
+     https
+ }
+```
+
+  * Add `Sender` struct
+    
+    We will add an implementation to the struct with a single function,
+    `try_payjoin`. The function will take a `payjoin::Uri` as an input and
+    attempt to perform a payjoin transaction.
+    
+```rust
+struct Sender;
+
+impl Sender {
+    pub async fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+    }
+}
+```
+
+
+In the next few steps, we will add the core implementation for the sender inside
+`try_payjoin` function.
+
+* Extract the parameters from the payjoin URI
+
+  Here we extract the amount the reciever is requesting and the address they
+  wish to recieve to. The URI also includes the endpoint we will make the
+  request to, and that is extracted in the `RequestContext::from_psbt_and_uri`
+  function and returned as part of the request data.
+
+```rust
+impl Sender {
+    pub async fn try_payjoin(pj_uri: payjoin::Uri<'static, NetworkChecked>) -> Txid {
+        let sender_wallet = get_client(Some("sender"));
+        let amount_to_send = pj_uri.amount.unwrap();
+        let receiver_address = pj_uri.address.clone();
+    }
+}
+```
+
+* Construct an "Original PSBT" paying `.amount` to `.address` as specified in
+  the URI.
+
+  First we add the reciever address to the outputs list along side the request
+  amount. Then we create a funded psbt with options set to specify `fee_rate`
+  and other paramters.
```

Comment:
explain why we're making an original psbt and how it's just a fallback in case we disappear

Reply:
sorry, can u please explain further? which side would disappear?